bedrock/tabzilla/templates/tabzilla/tabzilla.js

```diff
@@ -351,29 +343,13 @@ var Tabzilla = (function (Tabzilla) {
     };
     Infobar.prototype.trackEvent = function (action, label, value,
                                              nonInteraction, callback) {
-        if (typeof(_gaq) !== 'object') {
-            if (callback) {
-                callback();
-            }
-
+        if (typeof(window.gaTrack) !== 'function') {
```

Comment:
Hm, I'm not sure removing GA entirely from Tabzilla is viable. I'll check with the team tomorrow.